Makefile

```diff
@@ -2,6 +2,8 @@ BINARY=bc50d971-d4c9-42c4-82cb-343fb7f37896
 
 -include $(TA_DEV_KIT_DIR)/mk/ta_dev_kit.mk
 
+CFLAGS+=-fno-strict-aliasing
```

Comment:
Please move this into `sub.mk` as:
```
cflags-y += -fno-strict-aliasing
```

Reply:
You mean that if it can warn about it, it should also be able to prevent potential problems? That would be great, but apparently it's not that simple.